lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -496,16 +527,29 @@ deleteCheckpoints wid = do
     deleteWhere [CheckpointTableWalletId ==. wid]
     deleteState @s wid -- clear state
 
--- | Delete transactions that belong to a wallet and aren't referred to by
--- either Pending or TxMeta.
-deleteLooseTransactions
+-- | Delete unused TxMeta values for a wallet.
```

Comment:
Comment is a bit of a lie here :grimacing:  ... the function doesn't care much about whether the meta is used or unused, it just cleans them all.

Reply:
fixed. also take care of indenting in this file. Some functions had 2-character indents in this file